### PR TITLE
coreclr path being set incorrectly when dotnet_home env var isn't present

### PR DESCRIPTION
**Pull request by brthor**
cc @anurse @davidfowl @Sridhar-MS

### CODE REVIEW

File: src/dotnet/commands/dotnet-run/RunCommand.cs [hunk 1]
```diff
@@ -133,21 +133,22 @@ private int RunExecutable()
...
-            string runtime = Environment.GetEnvironmentVariable("DOTNET_HOME");
-            if (string.IsNullOrEmpty(runtime))
+            string dotnetHome = Environment.GetEnvironmentVariable("DOTNET_HOME");
+            if (string.IsNullOrEmpty(dotnetHome))
             {
                 // Use the runtime deployed with the tools, if present
-                var candidate = Path.Combine(AppContext.BaseDirectory, "..", "runtime");
+                var candidateBase = Path.Combine(AppContext.BaseDirectory, "..");
+                var candidate = Path.Combine(candidateBase, "runtime");
                 if (File.Exists(Path.Combine(candidate, Constants.LibCoreClrName)))
```
Comment (anurse) [inline src/dotnet/commands/dotnet-run/RunCommand.cs]: `libcoreclr` is actually in `DOTNET_HOME/runtime/coreclr`, not `DOTNET_HOME/runtime`

**Comment by anurse** [comment]
Path is slightly wrong, otherwise looks good to me :shipit:

File: src/dotnet/commands/dotnet-run/RunCommand.cs (hunk 1 above)
Comment (brthor) [inline src/dotnet/commands/dotnet-run/RunCommand.cs]: good catch, will make the change.

**Comment by brthor** [comment]
@dotnet-bot test this please

**Comment by livarcocc** [comment]
This is awesome. FInally. :shipit:

**Comment by brthor** [comment]
@dotnet-bot test this please

**Comment by brthor** [comment]
@dotnet-bot test CentOS7.1 Debug Build please

**Comment by brthor** [comment]
@dotnet-bot test CentOS7.1 Release Build please

**Pull request merged**